src/test/java/core/basesyntax/service/fruittransactionparser/FruitTransactionParserImplTest.java

```diff
@@ -0,0 +1,115 @@
+package core.basesyntax.service.fruittransactionparser;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+public class FruitTransactionParserImplTest {
+    private static final String TITLE =
+            "fruit,quantity" + System.lineSeparator();
+    private List<String> rowList;
+    private List<FruitTransaction> fruitTransactions;
+    private FruitTransactionParser fruitParser;
+
+    public FruitTransactionParserImplTest() {
+        this.rowList = new ArrayList<>();
+        this.fruitTransactions = new ArrayList<>();
+        this.fruitParser = new FruitTransactionParserImpl();
+    }
+
+    @Before
+    public void setUp() throws Exception {
+        rowList.add(TITLE);
+    }
+
+    @Test
+    public void validString_Ok() {
+        String fruitLine = "s,banana,100";
+        rowList.add(fruitLine);
+        FruitTransaction fruit = new FruitTransaction(FruitTransaction.Operation.SUPPLY,
+                "banana",100);
+        fruitTransactions.add(fruit);
+        List<FruitTransaction> expected = fruitTransactions;
+        List<FruitTransaction> actual = fruitParser.parseToFruitTransactions(rowList);
+        assertEquals(expected.toString(),actual.toString());
+    }
+
+    @Test
+    public void nullString_NotOk() {
```

Comment:
I think it's better to check for exceptions in negative tests, like 
@Test(expected = exception.class)

Don't need use fail method